### PR TITLE
Implement tileset enpoint

### PR DESCRIPTION
**Pull request by dgaubert**

### CODE REVIEW

File: app/services/carto/bigquery_tilesets_service.rb [hunk 1]
```diff
@@ -67,6 +69,33 @@ def count_tilesets(dataset_id)
...
+          public = true
+        end
+      end
+
+      metadata = tileset_metadata(tileset_id)
+      service_account = JSON.parse(@service_account)
+
+      {
+        id: tileset_id,
+        owner: service_account['client_email'],
```
Comment (jgoizueta) [inline app/services/carto/bigquery_tilesets_service.rb:91]: I don't really understand why the service account is the owner (now that we give the permissions to use the UDFs to the CARTO account email) but what do I know? 🤷  I'll trust you on this.

**Review by jgoizueta — APPROVED** [review]
LGTM

File: app/services/carto/bigquery_tilesets_service.rb (hunk 1 above)
Comment (dgaubert) [reply app/services/carto/bigquery_tilesets_service.rb:91]: Probably is wrong, but there is no way to get the owner of a table AFAIK. I tried by querying table metadata and fetching table policy with no luck.

Comment (dgaubert) [reply app/services/carto/bigquery_tilesets_service.rb:91]: I'm still researching how to achieve it properly.

**Pull request merged**